Makefile

```diff
@@ -31,3 +32,7 @@ extract_translations: ## extract strings to be translated, outputting .po files
 	fi
```

Comment:
it will use the option @OmarIthawi , but this goes into another PR
* Upgrade requirements
* Use new options `--no-segment --merge-po-files`
* Remove `django.po` and the `text.po` symlink